<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VclientStart_Config</class>
 <widget class="QMainWindow" name="VclientStart_Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>7</y>
      <width>391</width>
      <height>235</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>381</width>
       <height>220</height>
      </rect>
     </property>
     <property name="title">
      <string>Start mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QRadioButton" name="radioButton_local">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>125</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>After startup into the local terminal interface</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_auto">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>30</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto launch desktop when system start.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_auto">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Terminal need to add to the host server subnet,this mode in order to login with default passwd,&lt;/p&gt;&lt;p&gt;By the desktop association name, automatically connect to the remote desktop&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_manual">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>150</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Boot client after system startup</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>51</x>
        <y>175</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Service Ip :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ip"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>245</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_buttom"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
